sequencer/src/network/mod.rs

```diff
@@ -0,0 +1,86 @@
+use std::fs::read_to_string;
+
+use anyhow::Result;
+use hotshot_types::message::Message;
+use libp2p::BootstrapNode;
+use persistence::NetworkConfig;
+
+use super::*;
+
+pub mod cdn;
+pub mod libp2p;
+
+/// The genesis network configuration. Overrides what is received from the orchestrator
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct GenesisNetworkConfig {
+    /// An optional list of bootstrap nodes (multiaddress and peer ID) to connect to
+    pub bootstrap_nodes: Option<Vec<BootstrapNode>>,
+}
+
+impl GenesisNetworkConfig {
+    /// Load the genesis network config from a file
+    pub fn load_from_file(path: String) -> Result<GenesisNetworkConfig> {
```

Comment:
Is this used anywhere, since we normally just get the network config as a subset of the genesis file?

Reply:
Nope, forgot to remove this when I switched things over. Thanks for catching it